Doc/library/zipfile.rst

```diff
@@ -400,6 +400,10 @@ ZipFile Objects
       a closed ZipFile will raise a :exc:`ValueError`.  Previously,
       a :exc:`RuntimeError` was raised.
 
+   .. versionadded:: 3.8
+      The *strict_timestamps* keyword argument allows zip files older
+      than 1/1/1980 at the cost of setting the year to 1980.
+
```

Comment:
Please use the international and unambiguous "ISO format" to refer to dates: `1980-01-01`.